src/Pages/hyunchanpark/Login/Login.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router';
+import './Login.scss';
+
+const regObj = {
+  emailRegExp:
+    /[a-zA-Z0-9.-_+!]+@[a-zA-Z0-9]+\.[a-zA-Z0-9]{2,}(?:.[a-zA-Z0-9]{2,3})?/,
+  passwordRegExp: /[a-zA-Z0-9]{5,100}/,
+};
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { userId: '', userPw: '' };
+  }
+
+  validate = (value, regExp) => {
+    const reg = new RegExp(regExp);
+    let isValid = false;
+    if (reg.test(value)) isValid = true;
+    return isValid;
```

Comment:
```suggestion
return reg.test(value)
```
test method의 return값 자체가 boolean이기 때문에 위와 같이 간결하게 표현 가능해보입니다!

Reply:
리뷰 반영했습니다.